docs/content/how-to/writing-tables.md

```diff
@@ -234,6 +234,47 @@ For more information of drop-partition, see
 
 {{< /tabs >}}
 
+## Updating tables
+Currently, Paimon supports updating records by using `UPDATE` in Flink. You can perform `UPDATE` in Flink's `batch` mode.
```

Comment:
Use  "in Flink 1.17 and later versions."?

Reply:
resolved.